scenarios/reproducers/va-common.yml

```diff
@@ -0,0 +1,67 @@
+---
+# This file groups common parameters for Architecture based deployment.
+# It is *automatically* included in any run having cifmw_architecture_scenario
+# parameter set.
+# You can of course manually pass it, it won't make any difference.
+#
+# Any of those parameter might be overridden from your own parameter file!
+#
+# This file should NOT be used for the CRC based deployment (3-nodes.yml).
```

Comment:
Can we use an assert checking for something like `cifmw_rhol_crc_config` which would be defined if 3-nodes.yml was being used to enforce this?

Reply:
including the file shouldn't break things either. Just... "kind of useless data for your 3-nodes". It's "should", not "must" :).